Dockerfile

```diff
@@ -18,6 +18,12 @@ RUN DEBIAN_FRONTEND=noninteractive wget -qO - https://research.cs.wisc.edu/htcon
     && apt-get update -y \
     && apt-get install -y condor
 
+# install jars
+# perhaps we should have test and prod dockerfiles to avoid jars and mongo installs in prod
```

Comment:
++ 
That's a fairly common paradigm

Reply:
https://github.com/kbase/execution_engine2/issues/310